jenkins/ci.suse.de/macros.yaml

```diff
@@ -23,113 +23,6 @@
-          if [[ ${ROOT_BUILD_CAUSE} != "MANUALTRIGGER" ]]; then
-            trap jtsync_trap EXIT ERR
-          fi
-
-          # automation bootstrapping
-          if ! [ -e ${automationrepo}/scripts/jenkins/update_automation ] ; then
-              rm -rf ${automationrepo}
-              curl https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/jenkins/update_automation | bash
-          fi
-          ${automationrepo}/scripts/jenkins/update_automation
-
-          OBS_TYPE=${PROJECTSOURCE%%/*}
-          OBS_PROJECT=${PROJECTSOURCE##*/}
-
-          BUILD_DIST=SLE_12_SP4
-          OSC_BUILD_ARCH=x86_64
-
-          case $project in
-              Devel:Cloud:6*)
-                  BUILD_DIST=SLE_12_SP1
-                  ;;
-              Devel:Cloud:7*)
-                  BUILD_DIST=SLE_12_SP2
-                  ;;
-              Devel:Cloud:8*)
-                  BUILD_DIST=SLE_12_SP3
-                  ;;
-          esac
-
-
-          case $OBS_TYPE in
-            OBS) OSCAPI="https://api.opensuse.org"
-                OSC_BUILD_DIST=$BUILD_DIST
-                ;;
-            IBS) OSCAPI="https://api.suse.de"
-                OSC_BUILD_DIST=$BUILD_DIST
-                ;;
-            *)   echo "This jenkins instance only interacts with OBS or IBS."
-                exit 1
-                ;;
-          esac
-
-          # remove accidentally added spaces
-          component=${component// /}
-          OBS_PROJECT=${OBS_PROJECT// /}
-
-          if [ -z "$component" ] ; then
-              echo "Error: Variable component is unset."
-              exit 1
-          fi
-
-          export OSCAPI
-          export OSC_BUILD_DIST
-          export OSC_BUILD_ARCH
-
-          export JHOME=/home/jenkins
-          export BS_CHECKOUT=$JHOME/${OBS_TYPE}_CHECKOUT/$OBS_PROJECT
-          export OSC_BUILD_ROOT=$JHOME/buildroot
-
-          mkdir -p ~/.obs
-          for i in incoming repo repourl
-          do
-              mkdir -p $JHOME/obscache/tar_scm/$i
-          done
-          echo "CACHEDIRECTORY=\"$JHOME/obscache/tar_scm\"" > ~/.obs/tar_scm
-
-          mkdir -p "$BS_CHECKOUT"
-          cd "$BS_CHECKOUT"
-
-          rm -rf "$component"
-          osc -A $OSCAPI co -c "$OBS_PROJECT" "$component"
-
-          [ -d "$component" ] || ( echo "Error: Component $component does not exist (yet) or has been removed."  ; exit 1 )
-          cd "$component"
-
-          grep -q "<linkinfo" .osc/_files || exit 2
-
-          ${automationrepo}/scripts/jenkins/track-upstream-and-package.pl
-          ret=$?
-
-          # only enable jtsync when build is not manually triggered
-          if [[ ${ROOT_BUILD_CAUSE} != "MANUALTRIGGER" ]]; then
-            trap - EXIT ERR
-            $jtsync --ci suse --matrix ${main_job_name},${project},${BUILD_NUMBER} $ret || :
-          fi
-          exit $ret
+          ~/github.com/SUSE-Cloud/automation/scripts/jenkins/track-upstream.sh
```

Comment:
This should be track-upstream-internal.sh, right?

Reply:
Thx. Fixed.